src/test/java/com/gui/HomePageTest.java

```diff
@@ -5,19 +5,44 @@
 import com.gui.pages.desktop.ProductsPage;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 
-public class HomePageTest  {
-    WebDriver driver = new ChromeDriver();
+public class HomePageTest extends AbstractTest {
 
     @Test
     public void test(){
-        driver.get("https://automationteststore.com/");
-        HomePage homePage =new HomePage(driver);
+        HomePage homePage =new HomePage(getDriver());
         ProductsPage productsPage= homePage.searchKeyWord("shirt");
         productsPage.printAllItems();
     }
 
+    @Test(groups = "group1")
+    public void testCurrencyChange() {
+        HomePage homePage =new HomePage(getDriver());
+        homePage.clickCurrencySelector();
+        homePage.clickItemInCurrencySelector(0); //chooses the pound currency
+        Assert.assertTrue(homePage.isCurrencyInCartCorrect("£"));
+    }
+
+    @Test(groups = "group1")
+    public void testCartItemCount(){
+        HomePage homePage =new HomePage(getDriver());
+        homePage.clickAddToCartButton();
+        homePage.clickAddToCartButton();
+        homePage.clickAddToCartButton();
+        Assert.assertTrue(homePage.isCartItemCountCorrect("3"));
```

Comment:
The message in the assertion should provide helpful debugging information in case the test fails.